src/commands/addDataPlaneApis.ts

```diff
@@ -21,16 +21,16 @@ export namespace ConnectDataPlaneApi {
         if (!tenantid) {
             return;
         }
-        ConnectDataPlaneApi.sendDataPlaneApiTelemetry(endpointUrl, clientid, tenantid);
+        ConnectDataPlaneApi.sendDataPlaneApiTelemetry(endpointUrl, clientid, tenantid, TelemetryEvent.addDataPlaneApiFromInput);
```

Comment:
As wee discussed, it is better to distinguish in property not event name...

Reply:
set same event `dataPlane.addApiInstance` with different prop `dataPlaneApiFromInput` and `dataPlaneApiFromDeepLink`